### DISCUSSION TITLE
[Solved] Help! Rewards gone after chia GUI reinstall

### DISCUSSION THREAD

**Discussion by Proletche** [Chia-Network/chia-blockchain#5571] [Support]
Hi all,

Can anyone assist me with the following?
So i had troubles with the GUI syncing because I was changing the ports at the config.yaml file. After that i could not get the GUI to start syncing. The next thing i did was to re-install the Chia client - no change.
I had to delete the .chia folder  from "C:\Users\username\.chia\" and reinstall the client again.
The wallet was already there (strangely) even though i would have to re-introduce it.
The client started syncing from scratch which i kind of expected, but what i did not expect was my rewards to be gone. My initial thought was, OK, it should sync up, get up to date with network, probably find the block that was rewarded to my wallet and all will be good - Nope.
After a full sync, my reward still shows i haven't wont any chia, no transaction history, nothing.
Does anyone have any idea how is this possible.

My understanding is that, if i lose access to the PC with the client, i still have access to my wallet/funds with the seed phrase, right? 

I just can't believe that this has happened. Any ideas/suggestions would be greatly appritiated.

Thank you all.

**Comment by SackofFluor** [comment]
well firstly try to find your keys.

With your public wallet key you should be able to search on chianetwork whether you received.

Is it possible that, by chance you farmed with another wallet?
I had the problem, that the first wallet I created left the "receiving" address in the farmer.
So when I plotted with a new wallet the reward I earned was send to my "old wallet"...

**Reply by Proletche** [reply]
Thanks for the reply mate.

I've only ever created one account/key, so i dont think i even have the possibility of mixing those up (thank god :)) 

However, i just realized that my wallet it still not synced. Didn't realized that this had a syncing process too. Lets just hope all will be OK after its done (fingers crossed)

Lastly, i have compared the keys i had a notepad with my public key -farmer and -pool and they match... so thats good!

I'll hope this is just my silly oversight, and not an actual issue. Will report back of course.

Thanks again!

**Reply by XIII39** [reply]
Your funds will be on the blockchain.. you can go to https://www.chiaexplorer.com/blockchain/address/[WALLET ADDRESS] and see what's what.. Once your wallet syncs you'll have your funds back.. You can also see what height you received your coins so you know just how far you have to sync to see them in your wallet..

There are times that 100% of your coins are not on the blockchain explorer but it's not because it's not there, it's because there are multiple addresses attached to your private key so sometimes the small change gets attached to the other wallet addresses..

**Reply by Proletche** [reply]
yes! thats awesome info mate, bunch of thanks!

I just looked up my address on the blockchain and its all good.

Shows that i've received the reward at block: 247,828 and my wallet is currently at: 236,982

I guess its just a mater of time and sync.

Sorry for my freak out everyone... and thanks many thanks!

**Reply by XIII39** [reply]
No worries.. I know that sinking feeling in your stomach when you load your wallet and it says Total Balance = 0

I also like to keep an eye on what my farm pool has done on the blockchain.. no real reason other than it's nice to look at.. haha..

https://www.chiaexplorer.com/blockchain/pool/[ENTER_YOUR_PUBLIC_POOL_KEY]

**Reply by Proletche** [reply]
Everything is OK!!! 

Thanks a lot folks, it means a lot to have a support in a moment of total panic!

@XIII39, haha exactly!!! Very dreadful feeling.

**Comment by BrandtH22** [comment]
@Proletche , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
